server.js

```diff
@@ -47,9 +64,33 @@ app.post('/registration', async (request, response) => {
     }
   } catch (error) {
     console.error(error);
-    response.status(400).json(error.errors[0].message);
-  } 
+    response.status(400).send(error.errors[0].message);
+  }
+});
+
+app.post('/login', async (req, res) => {
+  console.log('Login endpoint hit. Request body:', req.body)
+  const { username, password } = req.body;
+
+  const user = await User.findOne({ where: { username: username } });
+  if (!user) {
+    return res.status(400).json({ error: 'User not found' });
+  }
+
+  const validPassword = await bcrypt.compare(password, user.password);
+  if (!validPassword) {
+    return res.status(400).json({ error: 'Invalid password' });
+  }
+
+  const token = jwt.sign({ id: user.id }, config.JWT_KEY, { expiresIn: '1h' });

```

Comment:
I was about to ask and forgot that `1s` seemed a bit short for expiration time 😁
I suppose you figured it out by yourself.

Reply:
Ah yes I was trying to test if the token expiration was working 👍